spec/features/workflow/withdraw_without_managing_editor_permission_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.feature "Withdraw without managing editor permission" do
   background do
-    given_i_dont_have_the_managing_editor_permission
+    given_i_am_not_a_managing_editor
```

Comment:
Do we need this step in the first place?

Reply:
No, suppose we don't since we don't do anything explicit anymore - will snip.